detail/fleet.cattle.io.gitrepo.vue

```diff
@@ -22,7 +26,49 @@ export default {
     },
   },
 
+  data() {
+    return {
+      allFleet:   [],
+      allBundles: [],
+    };
+  },
+
+  computed: {
+    harvesterClusters() {
+      const harvester = {};
+
+      this.allFleet.forEach((c) => {
+        if (isHarvesterCluster(c)) {
+          harvester[c.metadata.name] = c;
+        }
+      });
+
+      return harvester;
+    },
+
+    bundleCounts() {
+      return resourceCounts(this.$store, FLEET.BUNDLE);
+    },
+    bundles() {
+      const harvester = this.harvesterClusters;
+
+      const bundles = this.allBundles.filter((bundle) => {
+        const targets = bundle.spec?.targets || [];
+
+        // Filter out any bundle that has one target whose cluster is a harvester cluster
+        if (targets.length === 1) {
+          return !harvester[targets[0].clusterName];
+        }
+
+        return true;
+      });
+
+      return bundles;
+    },
+  },
   async fetch() {
```

Comment:
If these dispatches/requests don't have a requisite to be in series, you should consider doing them all in parallel using `await allHash`. Will improve load time

https://github.com/rancher/dashboard/blob/master/pages/c/_cluster/fleet/index.vue#L19

Reply:
This is fixed I am not sure why its not showing up as outdated.